tests/test_app.py

```diff
@@ -197,118 +232,130 @@ def test_sim_sir():
     assert isinstance(model.infected, float)  # based off note in models.py
 
     # test the class-calculated attributes
     assert model.detection_probability == 0.125
     assert model.intrinsic_growth_rate == 0.12246204830937302
-    assert model.beta == 3.582761451679408e-07
+    assert model.beta == 4.21501347256401e-07
     assert model.r_t == 2.307298374881539
     assert model.r_naught == 2.7144686763312222
     assert model.doubling_time_t == 7.764405988534983
 
-    # test the things n_days creates, which in turn tests sim_sir, sir, and get_dispositions
-    assert len(model.raw_df) == param.n_days + 1 == 61
 
+def test_model_raw_start(model=MODEL, param=PARAM):
     raw_df = model.raw_df
+
+    # test the things n_days creates, which in turn tests sim_sir, sir, and get_dispositions
+
+    # print('n_days: %s; i_day: %s; n_days_since: %s' % (param.n_days, model.i_day, model.n_days_since))
+    assert len(raw_df) == (param.n_days + model.n_days_since + 1) == 105
+
     first = raw_df.iloc[0, :]
     second = raw_df.iloc[1, :]
-    last = raw_df.iloc[-1, :]
 
     assert first.susceptible + first.infected + first.recovered == param.population
-    assert last.susceptible + last.infected + last.recovered == param.population
-
     assert first.susceptible == 460000.0
     assert round(second.infected, 0) == 43735
-
-    assert round(last.susceptible, 0) == 59497
+    assert list(model.dispositions_df.iloc[0, :]) == [-38, date(year=2020, month=2, day=19), 100.0, 40.0, 20.0]
     assert round(raw_df.recovered[30], 0) == 216711
 
-    assert list(model.dispositions_df.iloc[0, :]) == [0, 100.0, 40.0, 20.0]
-    assert [round(i, 0) for i in model.dispositions_df.iloc[60, :]] == [60, 1101.0, 441.0, 220.0]
+    d, dt, s, i, r = list(model.dispositions_df.iloc[60, :])
+    assert dt == date(year=2020, month=4, day=19)
+    assert [round(v, 0) for v in (d, s, i, r)] == [22, 1101.0, 441.0, 220.0]
+
+
+def test_model_raw_end(model=MODEL, param=PARAM):
+    raw_df = model.raw_df
+
+    last = raw_df.iloc[-1, :]
+    assert last.susceptible + last.infected + last.recovered == param.population
+    assert round(last.susceptible, 0) == 46683.0
 
-    # test that admissions are being properly calculated
-    cumulative_admits = model.admits_df.cumsum()
-    diff = cumulative_admits.hospitalized[1:-1] - (
-        0.05 * 0.05 * (raw_df.infected[1:-1] + raw_df.recovered[1:-1]) - 100
+
+def test_model_cumulative_census(model=MODEL):
+    # test that census is being properly calculated
+    raw_df = model.raw_df
+    admits_df = model.admits_df
+    df = pd.DataFrame({
+        "hospitalized": admits_df.hospitalized,
+        "icu": admits_df.icu,
+        "ventilated": admits_df.ventilated
+    })
+    admits = df.cumsum()
+
+    # TODO: is 1.0 for ceil function?
+    diff = admits.hospitalized[1:-1] - (
+        0.05 * 0.05 * (raw_df.infected[1:-1] + raw_df.recovered[1:-1]) - 1.0
     )
     assert (diff.abs() < 0.1).all()
```

Comment:
This used to be - 100 instead of -1.0. Why is there a difference, and a consistent difference at all?